lib/createGltf.js

```diff
@@ -194,6 +198,31 @@ function getTexture(gltf, texture) {
     };
 }
 
+function cloneMaterial(material, removeTextures) {
+    if (material === null || typeof material !== 'object') {
```

Comment:
This is to handle `typeof null` being "object".

I might remove it though since a material will never have a null property, and this function is already somewhat specialized for cloning materials.